pkg/pillar/hypervisor/kvm.go

```diff
@@ -358,14 +358,6 @@ const qemuDiskTemplate = `
 {{end}}`
 
 const qemuNetTemplate = `
-[device "pci.{{.PCIId}}"]
```

Comment:
I'm not sure I prefer the new content of the qemuNetTemplate compared to the previous version. I understand the reasoning behind separating the PCI root port, but we still need to create one for each network device and can't skip it. To me, it seems reasonable to include the root port as part of this template, but I recognize this is a subjective opinion.